doc/docs/master/how-tos/basic-data-operations/sql-ingest.md

```diff
@@ -1,251 +1,232 @@
+
+### Example: Snowflake
+
+In this example, we are leveraging Snowflake's support for queries traversing [semi-structured data](https://docs.snowflake.com/en/user-guide/semistructured-concepts.html#:~:text=Snowflake%20provides%20native%20support%20for,fast%20and%20efficient%20SQL%20querying.) (here, JSON). 
+
+
+1. Create a secret with your password named `snowflakeSecret`.
+2. Create a [Snowflake specific](#snowflake) database connection URL using the following details:
+      - **Protocol**: `snowflake`
+      - **Username**: `username`
+      - **Host**: `VCNYTW-MH64356` (account name or locator)
+      - **Database**: `SNOWFLAKE_SAMPLE_DATA` 
+      - **Schema**: `WEATHER`
+      - **Warehouse**: `COMPUTE_WH`
+      ```
+      snowflake://username@VCNYTW-MH64356/SNOWFLAKE_SAMPLE_DATA/WEATHER?warehouse=COMPUTE_WH
+      ```
+3. Build query for the table `DAILY_14_TOTAL` using information from column `V`.  
+    ```sql
+    select T, V:city.name, V:data[0].weather[0].description as morning, V:data[12].weather[0].description as pm FROM DAILY_14_TOTAL LIMIT 1
+    ```
+4.  Define the pipeline spec by populating all of the parameter values:
+   
+   ```shell
+      pachctl update pipeline --jsonnet https://raw.githubusercontent.com/pachyderm/pachyderm/{{ config.pach_branch }}/src/templates/sql_ingest_cron.jsonnet \
+      --arg name=mysnowflakeingest \
+      --arg url="snowflake://username@VCNYTW-MH64356/SNOWFLAKE_SAMPLE_DATA/WEATHER?warehouse=COMPUTE_WH" \
+      --arg query="select T, V:city.name, V:data[0].weather[0].description as morning, V:data[12].weather[0].description as pm FROM DAILY_14_TOTAL LIMIT 1" \
+      --arg hasHeader=true \
+      --arg cronSpec="@every 30s" \
+      --arg secretName="snowflakeSecret" \
+      --arg format=json
+   ```
+5. Run the command.
+
+---
 
-          ```shell
-          pachctl update pipeline --jsonnet https://raw.githubusercontent.com/pachyderm/pachyderm/{{ config.pach_branch }}/src/templates/sql_ingest_cron.jsonnet  \
-          --arg name=mysnowflakeingest \
-          --arg url="snowflake://username@VCNYTW-MH64356/SNOWFLAKE_SAMPLE_DATA/WEATHER?warehouse=COMPUTE_WH" \
-          --arg query="select T, V:city.name, V:data[0].weather[0].description as morning, V:data[12].weather[0].description as pm FROM DAILY_14_TOTAL LIMIT 1" \
-          --arg hasHeader=true \
-          --arg cronSpec="@every 30s" \
-          --arg secretName="snowflakesecret" \
-          --arg format=json
-          ```
+## How Does This Work?
 
+SQL Ingest's Jsonnet pipeline spec, [**`sql_ingest_cron.jsonnet`**](https://github.com/pachyderm/pachyderm/blob/{{ config.pach_branch }}/src/templates/sql_ingest_cron.jsonnet), creates all of the following:
 
-!!! Note
-    `pachctl update pipeline` will create pipelines if none exist, or update your existing pipelines otherwise.
+- **1 Input Data Repo**: Used to store timestamp files at the cronSpec's set interval rate (`--arg cronSpec="pullInterval" \`) to trigger the pipeline.
```

Comment:
Are the `**1` supposed to be auto incrementing? In the docs, it's all 1s.

Reply:
> Are doc/docs/master/how-tos/basic-data-operations/sql-ingest.md and doc/docs/2.3.x/how-tos/basic-data-operations/sql-ingest.md supposed to be exactly the same? If so, is there a way to not duplicate them?

Right now 2.3 and master are exactly the same; we draft future versions in master and then migrate them to their own version directories as they get released. I think master can't actually be displayed publicly, so any non-master changes tend to be the ones to focus on.